pal/system-info/web/system-info.ts

```diff
@@ -198,6 +198,26 @@ class SystemInfo extends EventTarget {
         const supportMouse = document.documentElement.onmouseup !== undefined || EDITOR;
         // NOTE: xr is not totally supported on web
         const supportXR = typeof (navigator as any).xr !== 'undefined';
+        // refer https://stackoverflow.com/questions/47879864/how-can-i-check-if-a-browser-supports-webassembly
+        const supportWasm = (() => {
+            // iOS 15.4 has some wasm memory issue, can not use wasm for bullet
+            const isiOS15_4 = (this.os === OS.IOS || this.os === OS.OSX) && /(OS 15_4)|(Version\/15.4)/.test(window.navigator.userAgent);
```

Comment:
But why check OSX too?

Reply:
it's a history bug fix, it seams that on safari, there is some memory issue

Reply:
But the variable name is `isiOS15_4 `.

Reply:
yes, it means on safari iOS15.4

Reply:
updated here https://github.com/cocos/cocos-engine/pull/14974/commits/0e1101f954a3a3bd23a1ba0f470516c6613bb4b0